proposals/nnnn-actors.md

```diff
@@ -594,31 +640,21 @@ extension BankAccount {
 }  
 ```
 
-Actors are similar to classes in all respects independent of isolation: actor types can have `static` and `class` methods, properties, and subscripts. All of the attributes that apply to classes apply to actors in much the same way, except where those semantics conflict with actor isolation. An actor type satisfies an `AnyObject` requirement.
+Actors are similar to classes in all respects independent of isolation: actor types can have `static` and `class` methods, properties, and subscripts. All of the attributes that apply to classes apply to actors in much the same way, except where those semantics conflict with actor isolation.
 
-### Non-isolated declarations
+### Actor protocol
 
-An instance method, computed property, or subscript of an actor may be annotated with `nonisolated`.  When a declaration is `nonisolated`, it (or its accessors) have a `self` parameter that is not `isolated`.
-
-A declaration may be declared to be non-isolated:
+The actor protocol abstracts over all types that are actors. Each actor type implicitly conforms to `Actor`, but no other kind of type (class, enum, struct, etc.) can conform to the `Actor` protocol.
```

Comment:
I take this to mean it is legal to say:

```
protocol Worker: Actor {}
```

for whatever reason, say it could assert that all workers are on some exact `WorkerThreadPool` executor etc... 

Worth spelling it out?

Reply:
Yeah, I'll be adding more here soon.

Reply:
Just saw the PR as well, fantastic :)